### ISSUE TITLE
Handle text overflow by setting flex

### ISSUE THREAD

**Issue by guidezpl** [flutter/friendlychat-steps#94]
https://codelabs.developers.google.com/codelabs/flutter/#6

If the text is long enough, it causes overflow. We should set to flex 1 with an Expanded

```
return Container(
  child: Row(
    crossAxisAlignment: CrossAxisAlignment.start,
    children: [
      Container(
        margin: const EdgeInsets.only(right: 16.0),
        child: CircleAvatar(child: Text(_name[0])),
      ),
// Need to wrap this Column in an Expanded()
      Column(
        crossAxisAlignment: CrossAxisAlignment.start,
        children: [
          Text(_name, style: Theme.of(context).textTheme.headline4),
          Container(
            margin: EdgeInsets.only(top: 5.0),
            child: Text(text),
          ),
        ],
      ),
    ],
  ),
);
```

**Comment by gozeloglu** [comment]
Hİ @guidezpl I looked at the code and I saw [this line](https://github.com/flutter/friendlychat-steps/blob/master/steps/step5_finishing_touches/main.dart#L65). Is that what you want to do, or are you still expecting fixing the bug?

**Comment by gozeloglu** [comment]
I sent a PR without waiting for your response 😄. I could not wait to make contribution to open source 😄 I hope my contribution resolves this issue and merge to  `master` branch.

**Comment by guidezpl** [comment]
Merged! Fixed by #96
